docs/_docs/persistence/snapshots.adoc

```diff
@@ -131,15 +134,18 @@ include::{javaCodeDir}/Snapshots.java[tags=create, indent=0]
 == Restoring From Snapshot
 
 Currently, the data restore procedure has to be performed manually. In a nutshell, you need to stop the cluster,
-replace persistence data and other files with the data from the snapshot, and restart the nodes. The detailed procedure
-looks as follows:
+replace persistence data and other files with the data from the snapshot, and restart the nodes. Prior to performing
+restore steps do not forget to cleanup the Write-ahead and Checkpoint files related to the current cluster. They may
+trigger unnecessary crash-recovery procedures when the cluster starts from a snapshot.
+
+The detailed procedure looks as follows:
 
 . Stop the cluster you intend to restore
 . Do the following on each node:
-    - Remove all the files and directories under the `$IGNITE_HOME/work/{node_id}` directory. Clean the
+    - Remove all the files and directories related to the `{nodeId}` under your `$IGNITE_HOME/work/` directory. Clean the
 link:link:persistence/native-persistence#configuring-persistent-storage-directory[`db/{node_id}`] directory separately if
 it's not located under the Ignite `work` dir.
-    - Copy the files of belonging to a node with the `{node_id}` from the snapshot into the `$IGNITE_HOME/work/{node_id}` directory.
+    - Copy the files of belonging to a node with the `{node_id}` from the snapshot into the `$IGNITE_HOME/work/` directory.
```

Comment:
A grammar mistake introduced by me earlier: 

"copy the files of belogning to a node" - remove the "of" preposition.

Reply:
Fixed.